app/controllers/api/v8/courses/submissions/last_hour_controller.rb

```diff
@@ -0,0 +1,41 @@
+module Api
+  module V8
+    module Courses
+      module Submissions
+        class LastHourController < Api::V8::BaseController
```

Comment:
This class should be added to ApidocsController so that the documentation is visible in documentation -endpoint